src/tracer.ts

```diff
@@ -197,6 +197,13 @@ export const trace =
   ({ token, debug, edgeHost, switchOff, stepFunction }) =>
   (userHandler) =>
   async (event, context, callback) => {
+    if (!isAwsEnvironment()) {
+      let handlerReturnValue = await promisifyUserHandler(userHandler, event, context);
+      handlerReturnValue = normalizeLambdaError(handlerReturnValue);
+      const cleanedHandlerReturnValue = removeLumigoFromStacktrace(handlerReturnValue);
+      const { err, data, type } = cleanedHandlerReturnValue;
+      return performPromisifyType(err, data, type, callback);
```

Comment:
why not simply:
`return userHandler(event, context)`
?

Reply:
what if it comes with callback?

Reply:
and if there is exception we need to remove lumigo from there

Reply:
of course, add the callback.
I don't want to run any logic here. Even not the remove-from-stacktrace IMO